tomviz/RecentFilesMenu.cxx

```diff
@@ -230,7 +230,7 @@ void RecentFilesMenu::dataSourceTriggered(QAction* actn, bool stack)
       if (!QFileInfo::exists(actn->iconText()) && !stack) {
         // If the user tried to open a recent file that no longer exists, remove
         // it from the recent files
-        qWarning() << "Error: file '" << actn->iconText() << "' does not exist.";
+        QMessageBox::warning(nullptr, "Error", QString("The file '%1' does not exist").arg(actn->iconText()));
```

Comment:
We should set the parent to ensure it shows up in the correct location over the parent window.